backend/libinput/keyboard.c

```diff
@@ -54,8 +54,8 @@ void handle_keyboard_key(struct libinput_event *event,
 	struct libinput_event_keyboard *kbevent =
 		libinput_event_get_keyboard_event(event);
 	struct wlr_event_keyboard_key wlr_event = { 0 };
-	wlr_event.time_sec = libinput_event_keyboard_get_time(kbevent);
 	wlr_event.time_usec = libinput_event_keyboard_get_time_usec(kbevent);
+	wlr_event.time_msec = (uint32_t)(wlr_event.time_usec / 1000);
```

Comment:
So we don't have to do this everywhere can you write a little helper function to convert?

Reply:
where do you want to put it?

Reply:
Somewhere in the libinput backend, maybe backend.c